index.html

```diff
@@ -3,12 +3,12 @@
   <head>
     <meta charset="utf8"/>
     <title>No Survival - The Game</title>
-    <link href="/styles/layout.css" rel="stylesheet"/>
```

Comment:
the point was not that it wasn't required for your setup, still no reason to make it public imo.
This was a way to not setup a local server for you ;)